.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+*                                    @MetaMask/devs
```

Comment:
This is out of date, but `@MetaMask/engineering` doesn't work. I suspect it may be because we need to update the permissions on this repo.